forge-gui/res/cardsfolder/upcoming/pull_of_the_mist_moon.txt

```diff
@@ -0,0 +1,16 @@
+Name:Pull of the Mist Moon
+ManaCost:1 W W 
+Types:Enchantment
+K:Kicker:1 U
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigExile | TriggerDescription$ When CARDNAME enters the battlefield, exile target nonland permanent an opponent controls until CARDNAME leaves the battlefield.
+SVar:TrigExile:DB$ ChangeZone | Origin$ Battlefield | Destination$ Exile | ValidTgts$ Permanent.nonLand+OppCtrl | TgtPrompt$ Select target nonland permanent an opponent controls | Duration$ UntilHostLeavesPlay
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self+kicked | Execute$ TrigChoose | TriggerDescription$ When CARDNAME enters the battlefield, if it was kicked, choose a nonland permanent in your hand. It perpetually gains "When this permanent enters the battlefield, exile target nonland permanent an opponent controls until this permanent leaves the battlefield."
+SVar:TrigChoose:DB$ ChooseCard | ChoiceZone$ Hand | Choices$ Card.nonLand+YouOwn | ChoiceTitle$ Choose a nonland card in your hand | Amount$ 1 | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | StaticAbilities$ PerpetualORing | Name$ Pull of the Mist Moon's Perpetual Effect | Duration$ Permanent | SubAbility$ DBCleanup
+SVar:PerpetualORing:Mode$ Continuous | Affected$ ChosenCard | AddTrigger$ ETBExile | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | SubAbility$ DBCleanup | Description$ The chosen card perpetually gains "When this permanent enters the battlefield, exile target nonland permanent an opponent controls until this permanent leaves the battlefield."
```

Comment:
You need to be careful with when you can use Defined syntax, in most cases like this it's not supported